bin/zopen-init

```diff
@@ -222,10 +214,63 @@ init()
       fi
     fi
   fi
+}
+
+determineStatsCollection()
+{
+  if isIBMHostname; then
+    isHostIBM=true
+  fi
+  if [ -n "$enableStats" ] && $enableStats; then
+    isCollectingStats=true
+  elif [ -n "$enableStats" ] && ! $enableStats; then
+    isCollectingStats=false
+  elif $isHostIBM; then
+    printAttention "Attention: Turning on usage statistics as your host is an IBM host. If this is incorrect, rerun with --noenable-stats"
```

Comment:
do we want ibmers to be able to override with ! $enableStats ?

Reply:
Is there a possibility that a third-party may get access to an ibm.com host and would not like there stats collected?

Reply:
Hm. maybe. that's a good reason for people on an IBM machine to be able to override.